test/force-auto.py.in

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python3
+
+# This script generates a BATS file to exercise "ch-image build --force"
+# across a variety of distributions. It's used by Makefile.am.
+#
+# About each distribution, we remember:
+#
+#   - base image name
+#   - config name it should select
+#   - scope
+#       standard: all tests in standard scope
+#       full: one test in standard scope, the rest in full
+#   - any tests invalid for that distro
+#
+# For each distribution, we test these factors:
+#
+#   - whether or not --force is given (2)
+#   - commands that (5)
+#       - don't need --force, and fail
+#       - don't need --force, and succeed
+#       - apparently need --force but in fact do not
+#       - really need --force, and preparation:
+#           - is already done by the base image
+#           - is not already done
+#
+# This yields 5×2 = 10 tests per distribution.
```

Comment:
I believe the way things are written there is a constraint which would skip the case  `needed, w/o --force, preprep` from being run so I think it is actually 9 tests per distribution. Am I reading this correctly?